markdown/usage/installation.md

```diff
@@ -31,125 +17,60 @@ java -version
 mv nextflow ~/bin/
 # OR system-wide installation:
 # sudo mv nextflow /usr/local/bin
 ```
 
-See [nextflow.io](https://www.nextflow.io/) for further instructions on how to install and configure Nextflow.
-
-## Install the pipeline
-
-### Automatic
-This pipeline itself needs no installation - nextflow will automatically fetch it from GitHub if `nf-core/YOUR_PIPELINE` is specified as the pipeline name.
-
-### Offline
-The above method requires an internet connection so that Nextflow can download the pipeline files. If you're running on a system that has no internet connection, you'll need to download and transfer the pipeline files manually:
-
-```bash
-wget https://github.com/nf-core/YOUR_PIPELINE/archive/master.zip
-mkdir -p ~/my-pipelines/nf-core/
-unzip master.zip -d ~/my-pipelines/nf-core/
-cd ~/my_data/
-nextflow run ~/my-pipelines/nf-core/YOUR_PIPELINE-master
-```
-
-To stop nextflow from looking for updates online, you can tell it to run in offline mode by specifying the following environment variable in your ~/.bashrc file:
+You can also install Nextflow using [Bioconda](https://bioconda.github.io/):
 
 ```bash
-export NXF_OFFLINE='TRUE'
+conda install -c bioconda nextflow
 ```
 
-### Development
-
-If you would like to make changes to the pipeline, it's best to make a fork on GitHub and then clone the files. Once cloned you can run the pipeline directly as above.
-
-
-## Pipeline configuration
-By default, the pipeline loads a basic resource configuration:
-
-```nextflow
-process {
-
-  cpus = { check_max( 2, 'cpus' ) }
-  memory = { check_max( 8.GB * task.attempt, 'memory' ) }
-  time = { check_max( 2.h * task.attempt, 'time' ) }
-
-  errorStrategy = { task.exitStatus in [143,137,104,134,139] ? 'retry' : 'terminate' }
-  maxRetries = 1
-  maxErrors = '-1'
-
-  // Process-specific resource requirements
-  withName: fastqc {
-    time = { check_max( 8.h * task.attempt, 'time' ) }
-  }
-  withName:toolX {
-    cpus = { check_max( 8, 'cpus' ) }
-    memory = { check_max( 8.GB * task.attempt, 'memory' ) }
-  }
-  withName: toolY {
-    cpus = { check_max( 10, 'cpus' ) }
-    memory = { check_max( 200.GB * task.attempt, 'memory' ) }
-    time = { check_max( 5.h * task.attempt, 'time' ) }
-  }
-}
-
-params {
-  // Defaults only, expecting to be overwritten
-  max_memory = 128.GB
-  max_cpus = 16
-  max_time = 240.h
-  igenomes_base = 's3://ngi-igenomes/igenomes/'
-}
-```
+We recommend using a personal installation of Nextflow where possible, instead of using a system-wide installation. This makes it easier to update.
 
-This uses a number of sensible defaults for process requirements and is suitable for running
-on a simple (if powerful!) local server.
+Updating nextflow is as simple as running `nextflow self-update`.
 
-Be warned of two important points about this default configuration:
+Once installed you will probably need to configure Nextflow to run on your system. For instructions, see [_Nextflow configuration_](configuration).
```

Comment:
Is this a link to something?
Did you forgot a `#`?

Reply:
It is a link, to a page called `configuration`. Should work I think.. Can make it `configuration.md` if you prefer so that it works on both GitHub and the website. We are not very consistent in this though..

Reply:
I see, can we do `configuration.md` then, if I remember well you remove `.md` from the pages and links?

Reply:
cf https://github.com/nf-core/nf-co.re/pull/259/files#diff-f18d7381b1d628ed6e8a7a66cb1dbb8cR29

Reply:
The `.md` isn't (currently) removed from the URL, but the page does load. For example: https://nf-co.re/usage/installation.md

I can probably add a rule to redirect if .md is specified though, so that the URL is kept clean in the browser.

Reply:
✅ Added new htaccess rules to do automatic redirection
✅ Uses GitHub friendly markdown filenames everywhere now, I think..